hpx/util/unwrapped.hpp

```diff
@@ -495,20 +507,66 @@ namespace hpx { namespace util
                     util::detail::pack<typename std::decay<Ts>::type...>
                 >::type type;
         };
-    }
-
-    ///////////////////////////////////////////////////////////////////////////
+    } // end namespace detail
```

Comment:
Actually when I think about this doxygen shoudln't hide the namespace when not specified, because my research about it didn't reveal any default hidden names.

Maybe we could also officially ignore symbols like namespaces named as `detail` per default:

```diff
diff --git a/cmake/templates/autodoc.doxy.in b/cmake/templates/autodoc.doxy.in
index 86861a6..1f5a011 100644
--- a/cmake/templates/autodoc.doxy.in
+++ b/cmake/templates/autodoc.doxy.in
@@ -13,4 +13,5 @@ XML_OUTPUT = @doxygen_output_file@
 OUTPUT_DIRECTORY = @doxygen_output_dir@
 GENERATE_LATEX = NO
 INPUT = @doxygen_inputs@
+EXCLUDE_SYMBOLS = detail
```
I could open a PR for this.